contracts/governance/Badger.sol

```diff
@@ -229,14 +253,30 @@ contract Badger is Ownable, ERC1155 {
         uint256 tokenId,
         string calldata uriId,
         bool transferable
-    ) public onlyOwner isValidString(uriId) {
+    ) public onlyOwner {
+        _createTokenTier(tokenId, uriId, transferable);
```

Comment:
Can't we just call this function inside of batchCreateTokenTiers instead of implementing internal function?

Reply:
Yes we could, but then we would check `onlyOwner` in every iteration of the loop. I wanted to save on that. Does that make sense?

Reply:
Actually yes, it makes sense